<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modify_book</class>
 <widget class="QMainWindow" name="modify_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>599</width>
      <height>570</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="ISBN_label">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>ISBN</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ISBN">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="name_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>书名</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="name"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="author_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>作者</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="author"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="image_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>封面路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="image"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="publish_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>出版社</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="publish"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="publish_date_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>出版日期</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="publish_date"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="price_label">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>价钱</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="price"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="price_label_4">
             <property name="font">
              <font>
               <family>Adobe Devanagari</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
             </property>
             <property name="text">
              <string>入库量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="num_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_28">
       <item>
        <widget class="QLabel" name="classification_label">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
         </property>
         <property name="text">
          <string>分类</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_29">
           <item>
            <widget class="QCheckBox" name="cla_0">
             <property name="text">
              <string>哲学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_1">
             <property name="text">
              <string>人文社科综合</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_2">
             <property name="text">
              <string>管理学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_3">
             <property name="text">
              <string>政治</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_4">
             <property name="text">
              <string>法律</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_5">
             <property name="text">
              <string>经济</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_30">
           <item>
            <widget class="QCheckBox" name="cla_6">
             <property name="text">
              <string>文化/教育</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_7">
             <property name="text">
              <string>语言/文学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_8">
             <property name="text">
              <string>艺术</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_9">
             <property name="text">
              <string>历史</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_10">
             <property name="text">
              <string>数学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_11">
             <property name="text">
              <string>物理</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_12">
             <property name="text">
              <string>化学/材料</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_31">
           <item>
            <widget class="QCheckBox" name="cla_13">
             <property name="text">
              <string>理工综合</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_14">
             <property name="text">
              <string>地球科学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_16">
             <property name="text">
              <string>生物</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_17">
             <property name="text">
              <string>医学/药学</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_18">
             <property name="text">
              <string>农业/水产</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_32">
           <item>
            <widget class="QCheckBox" name="cla_19">
             <property name="text">
              <string>工业技术</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_20">
             <property name="text">
              <string>计算机/网络</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_21">
             <property name="text">
              <string>环境</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cla_22">
             <property name="text">
              <string>其他</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="description_label">
         <property name="font">
          <font>
           <family>Adobe Devanagari</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(233, 232, 227,100%)</string>
         </property>
         <property name="text">
          <string>简介</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="description">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="submit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/*	qproperty-icon: url(./appIcons/location-1.png);
按钮普通态*/
QPushButton
{
	border:0.5px groove gray;
	border-radius:5px;
	padding:2px 4px;
	background-color:rgb(233 , 232 , 227);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:5px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:5px;
}</string>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>提交.png</normaloff>提交.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_return">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">/*	qproperty-icon: url(./appIcons/location-1.png);
按钮普通态*/
QPushButton
{
	border:0.5px groove gray;
	border-radius:5px;
	padding:2px 4px;
	background-color:rgb(233 , 232 , 227);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(255 , 255 , 255);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:5px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:5px;
}</string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>返回.png</normaloff>返回.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>901</width>
      <height>691</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/img/back.png)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
